client/src/pages/reports/Form.js

```diff
@@ -446,8 +446,7 @@ class ReportForm extends ValidatableFormWrapper {
 				<LinkTo person={person}/>
 			</td>
 			<td><LinkTo position={person.position} /></td>
```

Comment:
When a person has no position it will show unspecified for position, maybe it's better to add whenUnspecified="" ?